graylog2-web-interface/src/components/configurations/URLWhitelistInput.jsx

```diff
@@ -0,0 +1,122 @@
+// @flow
+import React, { useRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Input } from 'components/bootstrap';
+import URLWhitelistFormModal from 'components/configurations/URLWhitelistFormModal';
+import StoreProvider from 'injection/StoreProvider';
+import URLUtils from 'util/URLUtils';
+import UIUtils from 'util/UIUtils';
+
+const ToolsStore = StoreProvider.getStore('Tools');
+type Props = {
+  label: string,
+  handleFormEvent: (event: SyntheticInputEvent<EventTarget>) => void,
+  validationMessage: string,
+  validationState: string,
+  url: string,
+  labelClassName: string,
+  wrapperClassName: string,
+  formType: string
+}
+const URLWhitelistInput = ({ label, handleFormEvent, validationMessage, validationState, url, labelClassName, wrapperClassName, formType }: Props) => {
```

Comment:
Since the `URLWhitelistInput` will be used in many different forms across the product, I think it should live somewhere else. Since some shared inputs are already in `components/common`, I would suggest moving it there.